ironfish/src/network/messages/pooledTransactionsRequest.ts

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import bufio from 'bufio'
+import { TransactionHash } from '../../primitives/transaction'
+import { NetworkMessageType } from '../types'
+import { Direction, RpcNetworkMessage } from './rpcNetworkMessage'
+
+export class PooledTrasactionsRequest extends RpcNetworkMessage {
```

Comment:
Should be "transactions", there are a couple other places that will need to be refactored to make this work

```suggestion
export class PooledTransactionsRequest extends RpcNetworkMessage {
```

Reply:
good catch